src/test/java/uk/co/ramp/covid/simulation/ModelTest.java

```diff
@@ -120,19 +124,78 @@ public void modelsWithSameRNGSeedGiveSameResult() {
 
     @Test
     public void testLockdown() {
-        int population = 10000;
-        int nInfections = 10;
 
-        Model m = new Model()
+        int startLock = 30;
+        int endLock = 60;
+
+        //Run the model with no lockdown
+        Model m1 = new Model()
+                .setPopulationSize(population)
+                .setnInfections(nInfections)
+                .setnHouseholds(nHouseholds)
+                .setIters(nIter)
+                .setnDays(nDays)
+                .setRNGSeed(RNGSeed)
+                .setNoOutput();
+
+        List<List<DailyStats>> stats1 = m1.run();
+
+        //Re-run the model with partial lockdown
+        Model m2 = new Model()
                 .setPopulationSize(population)
                 .setnInfections(nInfections)
-                .setnHouseholds(3000)
-                .setIters(1)
-                .setnDays(90)
-                .setRNGSeed(42)
+                .setnHouseholds(nHouseholds)
+                .setIters(nIter)
+                .setnDays(nDays)
+                .setRNGSeed(RNGSeed)
                 .setNoOutput()
-                .setLockdown(1, 20, 2.0);
+                .setLockdown(startLock, endLock, 2.0);
+
+        List<List<DailyStats>> stats2 = m2.run();
+
+        //Check that there are fewer infections in the lockdown scenario
+        int inf1 = stats1.get(0).get(nDays - 1).getTotalInfected();
+        int inf2 = stats2.get(0).get(nDays - 1).getTotalInfected();
+        assertTrue("Unexpected more infections under lockdown", inf1 > inf2);
+
+        //Test that the total number of infections before lockdown
+        //is higher than during lockdown
+        int totInfBeforeLockdown = 0;
+        int totInfDuringLockdown = 0;
+        int totInfAfterLockdown = 0;
+        for (int i = 0; i < stats2.get(0).size(); i++) {
+            if (i < startLock) {
+                totInfBeforeLockdown += stats2.get(0).get(i).getTotalDailyInfections();
+            } else if (i < endLock) {
+                totInfDuringLockdown += stats2.get(0).get(i).getTotalDailyInfections();
+            } else {
+                totInfAfterLockdown += stats2.get(0).get(i).getTotalDailyInfections();
+            }
+        }
+        assertTrue("Unexpectedly fewer infections before lockdown", totInfDuringLockdown < totInfBeforeLockdown);
+        assertTrue("Unexpectedly fewer infections after lockdown", totInfDuringLockdown < totInfAfterLockdown);
+    }
 
-        List<List<DailyStats>> stats = m.run();
+    @Test
+    public void testMortality() throws IOException {
+        //Mortality and transmission rates are set to 100%
+        //Check that everyone is infected and progresses to death
+        ParameterReader.readParametersFromFile("src/test/resources/Test_full_mortality.json");

```

Comment:
Should we add a setter for this as well? That way it's clear in the test that they are set to 100%

Reply:
Added setters for all parameters required to give 100% mortality